### PR TITLE
Track2 pythonmd

### PR DESCRIPTION
**Pull request by 00Kai0**
@zikalino update other six services python.md.

### CODE REVIEW

File: specification/storage/resource-manager/readme.python.md [hunk 1]
```diff
@@ -12,11 +12,21 @@ python:
   no-namespace-folders: true
 ```
 
+These settings apply only when `--track2` is specified on the command line.
+
+``` yaml $(track2)
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+payload-flattening-threshold: 2
```
Comment (zikalino) [inline specification/storage/resource-manager/readme.python.md:20]: for track1 let's remove payload-flattening-threshold

**Review by zikalino — APPROVED** [review]

**Comment by lirenhe** [comment]
@00Kai0, SDK generation failed, could you double check the root cause?

**Comment by 00Kai0** [comment]
Hi @lirenhe ,we will split this pr and create new one.
This PR can be closed, thx.

**Comment by zikalino** [comment]
closing as it was separated in several smaller prs

**Pull request closed without merging**